README.md

```diff
@@ -41,7 +41,32 @@ Then run `yarn start` to start the app, using the proxy conf to reroute calls to
 
 The application will be available on http://localhost:4200
 
+### External services
 
+The application stores uploaded documents in Google Cloud Storage. This means that the application
+needs credentials (which are json files generated by Google) in order to work fine.
+
+Two separate accounts, and thus credentials files, are used:
+
+ - one for development
+ - one for production
+ 
+We expect to stay inside the "Always Free" tier, whatever the account is.
+
+Here are various interesting links regarding Google Cloud Storage (GCS):
+
+ - [https://cloud.google.com/storage/](The landing page of GCS)
+ - [https://cloud.google.com/storage/pricing](The pricing page of GCS). It describes the conditions 
```

Comment:
reverse it : [https://cloud.google.com/storage/pricing](The pricing page of GCS) -> [The pricing page of GCS](https://cloud.google.com/storage/pricing)